azure/packages/azure-client/src/test/types/validateAzureClientPrevious.generated.ts

```diff
@@ -6,12 +6,20 @@
  * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
  * Generated by fluid-type-test-generator in @fluidframework/build-tools.
  */
-import * as old from "@fluidframework/azure-client-previous";
-import * as current from "../../index";
-
-type TypeOnly<T> = {
-    [P in keyof T]: TypeOnly<T[P]>;
-};
+import type * as old from "@fluidframework/azure-client-previous";
+import type * as current from "../../index";
+
+
+// See 'build-tools/src/type-test-generator/compatibility.ts' for more information.
+type TypeOnly<T> = T extends number
+	? number
+	: T extends string
+	? string
+	: T extends boolean | bigint | symbol
+	? T
+	: {
+			[P in keyof T]: TypeOnly<T[P]>;
+	  };
```

Comment:
Example of 2: Regenerating type test to update boiler plate.  Skim past these as well.